tempus_fugit_minecraft/main.py

```diff
@@ -893,7 +1025,7 @@ def setup():
 
 
 def main():
-    window = Window(width=800, height=600, caption='Pyglet', resizable=True)
+    window = Window(width=window_width, height=window_height, caption='Tempus Fugit Minecraft', resizable=True)
```

Comment:
I suggest running the game in full screen at the start.
Look for issue #45

Reply:
Okay, I will consider doing that in a future pull request.